testing/catch_main.cpp

```diff
@@ -1,5 +1,5 @@
 #define CATCH_CONFIG_RUNNER
-#include <catch2/catch.hpp>
+#include <catch2/catch_all.hpp>
```

Comment:
IIRC there's a bunch of specific header files that don't pull in some of the more expensive headers. I might have a branch with these somewhere, but for now these are fine